goblint.opam.locked

```diff
@@ -50,12 +50,12 @@ depends: [
   "cpu" {= "2.0.0"}
   "csexp" {= "1.5.1"}
   "ctypes" {= "0.20.1"}
-  "dune" {= "3.6.1"}
-  "dune-build-info" {= "3.6.1"}
-  "dune-configurator" {= "3.6.1"}
-  "dune-private-libs" {= "3.6.1"}
-  "dune-site" {= "3.6.1"}
-  "dyn" {= "3.6.1"}
+  "dune" {= "3.7.1"}
```

Comment:
The CI fails because some more dune dependencies need their versions bumped.

Reply:
Should now be fixed in [85e6fdb](https://github.com/goblint/analyzer/pull/1251/commits/85e6fdbfcba3d42bf35b44f1c41171e958890451)